No matching version found for gulp@next

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for gulp@next
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

Node v11.4.0
NPM 6.4.1

--- a/packages/generator-frontend/app/index.js
+++ b/packages/generator-frontend/app/index.js
@@ -514,7 +514,7 @@ class PixelGenerator extends Generator {
       '@pixel2html/scripts-frontend',
       'del',
       'fs-path',
-      'gulp@next',
+      'gulp',
       'gulp-zip',
       'yargs',
     ]

